frontend/src/core/scaffold/scaffold.js

```diff
@@ -149,8 +149,15 @@ define(function(require) {
 			if (_.isObject(field.inputType)) {
 				fieldObject = _.extend(fieldObject, field.inputType);
 			}
+
 			scaffoldSchema[key] = fieldObject;
 
+			var editorAttrs = field.editorAttrs;
+
+			if (editorAttrs) {
+				scaffoldSchema[key].editorAttrs = editorAttrs;
+			}
+			
```

Comment:
Can this be moved to sit after the mammoth if statement (currently ends on line 175), so that `object` and `array` types can get involved in the fun too?